rootfs/apply.go

```diff
@@ -74,7 +75,7 @@ func ApplyLayer(snapshots snapshot.Snapshotter, mounter Mounter, rd io.Reader, p
 		chainID = identity.ChainID([]digest.Digest{parent, chainID})
 	}
 
-	return diffID, snapshots.Commit(chainID.String(), key)
+	return diffID, snapshots.Commit(ctx, chainID.String(), key)
 }
 
 // Prepare the root filesystem from the set of layers. Snapshots are created
```

Comment:
Sorry, can't comment on the exact line, `Prepare` should have a context argument.